runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -1,12 +1,14 @@
 ---
 name: "AMQP 1.0 JMS client - Apache Qpid JMS"
 metadata:
+  short-name: artemis
```

Comment:
An existing user specifying project creation with -Dextensions="artemis" (which isnt what the guide suggested, it uses -Dextensions="artemis-jms", perhaps since artemis-core also exists?) would currently get the artemis-jms extension. Is this so that folks after simply 'artemis' instead get the qpid-jms extension now?

It does feels a little odd to me that something not artemis-specific would be short-named artemis, when something that is artemis-specific also exists (in quarkus core no less, so this would be somewhat of an external grab hehe). It might seem nicer if its short name were qpid-jms or amqp-jms or something along those lines. Can there be multiple short-names?

Reply:
I prefer "amqp-jms".  It's important not to use "artemis" or "artemis-jms" because it feeds the incorrect notion that this is "a JMS client for Artemis".  It's really a JMS client for any server that supports AMQP.

Along those lines, I'd like the description to list several AMQP 1.0 servers that the extension is compatible with.

Reply:
I used it as a way to slightly privilege the search engine towards this extension. Not to make a short name for actually picking it. So maybe we should remove the short name then and let the brand be the winner?

Reply:
If it wasn't for influencing the auto usage during the project creation (or even if it was really), then yes I would remove it. I think it would be confusing and potentially troublesome later having it there.